components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -670,6 +719,62 @@ var conditionalAuthFunctions = $.parseJSON('<%=availableJsFunctionsJson%>');
     }
 
 
+    function showPopupDialog(message, title, windowHight, okButton, callback, windowWidth) {
```

Comment:
typo in height

Also is this different from what we have now with carbonUI?

Reply:
Fixed with e5b7774 and b71d7b6